recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts

```diff
@@ -774,6 +786,7 @@
 	status = "okay";
 
 	iommus = <&ipmmu_ds1 34>;
+	power-domains = <&sysc R8A7795_PD_A3VP>;
```

Comment:
At least you have to add a comment describing why do you put power-domain here the sdhi is not even belongs to.

Reply:
done